mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -244,7 +244,12 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 	METHOD method_5634 getSoundCategory ()Lnet/minecraft/class_3419;
 	METHOD method_5636 setYaw (F)V
 		ARG 1 yaw
-	METHOD method_5637 isTouchingWater ()Z
+	METHOD method_5637 isWet ()Z
+		COMMENT Returns whether this entity is touching water or is being rained on or is inside a bubble column, etc...
```

Comment:
```suggestion
		COMMENT Returns whether this entity is touching water, or is being rained on, or is inside a bubble column...
```
Note that you would typically use either `etc.` or `...`, but not both at once.

Reply:
Oh ok, sorry, wasn't aware that was different in English, my French point of view have taken over this one.

Reply:
Heh, would you look at that: https://www.projet-voltaire.fr/regles-orthographe/etc-ou-etc/

Reply:
oh, my bad... Didn't know that :/